docs/quickstarts/1_client_credentials.rst

```diff
@@ -197,17 +188,25 @@ Update `Startup` to look like this::
 
         public void Configure(IApplicationBuilder app)
         {
+            app.UseRouting();
+
             app.UseAuthentication();
+            app.UseAuthorization();
 
-            app.UseMvc();
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapControllers();
+            });
         }
     }
 
 
-``AddAuthentication`` adds the authentication services to DI and configures ``"Bearer"`` as the default scheme. 
-``UseAuthentication`` adds the authentication middleware to the pipeline so authentication will be performed automatically on every call into the host.
+* ``AddAuthentication`` adds the authentication services to DI and configures ``Bearer`` as the default scheme. 
+* ``UseAuthentication`` adds the authentication middleware to the pipeline so authentication will be performed automatically on every call into the host.
+* ``UseAuthorization`` adds the authorization middleware to make sure, our API endoint cannit be accessed by anonymous clients.
```

Comment:
typo on this line

> our API enoint cannit

our API endpoint can not

Reply:
Thanks!